index.js

```diff
@@ -44,21 +43,20 @@ prog
         }
 
         return getLocalDependencies()
-            .then((deps) => clone(deps, logger))
-            .catch((err) => logger.error(LOG.ERROR, err));
+            .then(deps => clone(deps, logger))
+            .catch(err => logger.error(LOG.ERROR, err));
     })
-
     .command('validate', 'Validates local .zel file.')
     .action((args, options, logger) => {
         getLocalDependencies()
-            .then((deps) => {
+            .then(deps => {
                 resolver
-                    .on('valid', (repo) => logger.info(LOG.VALID, repo.repoName))
-                    .on('invalid', (repo) => logger.error(LOG.INVALID, repo.repoName))
+                    .on('valid', repo => logger.info(LOG.VALID, repo.repoName))
+                    .on('invalid', repo => logger.error(LOG.INVALID, repo.repoName))
                     .validate(deps)
-                        .catch((err) => logger.error(LOG.ERROR, err));
+                    .catch(err => logger.error(LOG.ERROR, err));
```

Comment:
Arguably, then, every `then` and `catch` should start a new line with extra indentation since they're also returning a new Promise chain. 😉